modules/apps/info/info-api/src/main/java/com/liferay/info/item/NoSuchClassTypeException.java

```diff
@@ -14,10 +14,12 @@
 
 package com.liferay.info.item;
 
+import com.liferay.portal.kernel.exception.PortalException;
+
 /**
  * @author Jorge Ferrer
  */
-public class NoSuchClassTypeException extends Exception {
+public class NoSuchClassTypeException extends PortalException {
```

Comment:
@4lejandrito Why this change?
I was on purpose trying to keep the info-fw exceptions independent of PortalException which is a kitchen sink exception